docs/azure/sdk/includes/implement-user-assigned-identity.md

```diff
@@ -17,53 +20,47 @@ dotnet add package Microsoft.Extensions.Azure
 
 ### [NuGet Package Manager](#tab/nuget-package)
 
-Right-click your project in Visual Studio's **Solution Explorer** window and select **Manage NuGet Packages**. Search for **Azure.Identity**, and install the matching package. Repeat this process for the **Microsoft.Extensions.Azure** package.
+Right-click your project in the Visual Studio **Solution Explorer** window and select **Manage NuGet Packages**. Search for **Azure.Identity**, and install the matching package. Repeat this process for the **Microsoft.Extensions.Azure** package.
 
 :::image type="content" source="../media/nuget-azure-identity.png" alt-text="Install a package using the package manager.":::
 
 ---
 
-Azure services are accessed using specialized client classes from the various Azure SDK client libraries. These classes and your own custom services should be registered so they can be accessed via dependency injection throughout your app. In `Program.cs`, complete the following steps to register a client class and `DefaultAzureCredential`:
+Azure services are accessed using specialized client classes from the various Azure SDK client libraries. These classes and your own custom services should be registered for dependency injection so they can be used throughout your app. In `Program.cs`, complete the following steps to configure a client class for dependency injection and token-based authentication:
 
 1. Include the `Azure.Identity` and `Microsoft.Extensions.Azure` namespaces via `using` directives.
 1. Register the Azure service client using the corresponding `Add`-prefixed extension method.
-1. Pass an instance of `DefaultAzureCredential` to the `UseCredential` method.
-
-> [!NOTE]
-> For a user-assigned managed identity, make sure to assign the identity's `clientId` value to the `ManagedIdentityClientId` property on the `DefaultAzureCredentialOptions` object. This enables your code to discover the correct identity to use for authentication while running in azure.
-
-For example:
-
-```c#
-using Microsoft.Extensions.Azure;
-using Azure.Identity;
-
-builder.Services.AddAzureClients(clientBuilder =>
-{
-    clientBuilder.AddBlobServiceClient(
-        new Uri("https://<account-name>.blob.core.windows.net"));
-    clientBuilder.UseCredential(new DefaultAzureCredential(
-        new DefaultAzureCredentialOptions()
-        {
-            ManagedIdentityClientId = "<your-client-id>"
-        }));
-});
-```
+1. Pass an appropriate `TokenCredential` instance to the `UseCredential` method:
+    - Use `DefaultAzureCredential` when your app is running locally
+    - Use `ManagedIdentityCredential` when your app is running in Azure and configure either the client ID, resource ID, or object ID.
```

Comment:
As a reader, I may wonder where to find each of these 3 values. I suggest including a short blurb in each tab with that info.